hpx/components/containers/partitioned_vector/partitioned_vector_view.hpp

```diff
@@ -182,7 +176,7 @@ namespace hpx
         iterator end()
```

Comment:
Same question as above: should this expose `const` versions of `begin` and `end` as well?